TeamSnapSDK/SDK/DataTypes/Members/TSDKContactEmailAddress.m

```diff
@@ -10,10 +10,40 @@
 
 @implementation TSDKContactEmailAddress
 
-@dynamic contactId, linkContact;
+@dynamic label, isHidden, createdAt, isInvitable, invitationState, isInvited, isAccepted, receivesTeamEmails, contactId, updatedAt, invitationCode, teamId, email, memberId, linkContact, linkMember, linkTeam;
 
 + (NSString *)SDKType {
     return @"contact_email_address";
 }
 
++ (void)actionInvite:(NSArray *_Nonnull)contactEmailAddresses withConfiguration:(TSDKRequestConfiguration *_Nullable)configuration completion:(TSDKCompletionBlock _Nullable)completion {
+    if (contactEmailAddresses.count > 0) {
+        TSDKContactEmailAddress *firstMemberEmailAddress = [contactEmailAddresses objectAtIndex:0];
```

Comment:
Normally I would say use `[contactEmailAddresses firstObject]`, but you're gating on the previous line on this array being non-empty.